gravitee-apim-rest-api/gravitee-apim-rest-api-service/src/main/java/io/gravitee/rest/api/service/impl/ApiExportServiceImpl.java

```diff
@@ -180,6 +187,34 @@ private void mapCRDMembers(ExecutionContext executionContext, ArrayNode members)
         }
     }
 
+    private void prepareApiPagesToExport(ObjectNode pages) throws JsonProcessingException {
+        Iterator<Map.Entry<String, JsonNode>> iterator = pages.fields();
+
+        while (iterator.hasNext()) {
+            Map.Entry<String, JsonNode> pageJsonNode = iterator.next();
+            PageEntity page = objectMapper.treeToValue(pageJsonNode.getValue(), PageEntity.class);
+            if (
+                (
+                    PageType.MARKDOWN.name().equals(page.getType()) ||
+                    PageType.SWAGGER.name().equals(page.getType()) ||
+                    PageType.FOLDER.name().equals(page.getType())
+                ) &&
+                page.getSource() != null &&
+                "github-fetcher".equals(page.getSource().getType())
+            ) {
+                iterator.remove();
+            } else if (
+                PageType.SWAGGER.name().equals(page.getType()) &&
+                page.getSource() != null &&
+                "http-fetcher".equals(page.getSource().getType())
+            ) {
+                ((ObjectNode) pageJsonNode.getValue()).remove("content");
+            } else if (PageType.SYSTEM_FOLDER.name().equals(page.getType())) {
+                iterator.remove();
```

Comment:
If you remove the system folder from an API, you will loose the LINK pages that are used in the developer portal to display custom links